File: Lib/SchemaRoot.cs
using System;
using System.Linq;

namespace System.Data.EntitySchema {

  public class SchemaRoot {

    public string DescriptionFormat { get; set; } = "";

    public string TimestampUtc { get; set; } = "";
    public string SemanticVersion { get; set; } = "";

    public EntitySchema[] Entities { get; set; } = { };

    public RelationSchema[] Relations { get; set; } = { };

  }

}

File: js/src/schemaRoot.ts
import { EntitySchema } from './entitySchema';
import { KnownValueRange } from './KnownValueRange';
import { RelationSchema } from './relationSchema';

/**
 * (from 'FUSE-fx.ModelDescription')
 */
export class SchemaRoot {
  public descriptionFormat: string = '';

  public timestampUtc: string = '';

  public semanticVersion: string = '';

  public entities: EntitySchema[] = [];

  public relations: RelationSchema[] = [];

  public designerData: string = '';

  public knownValueRanges: KnownValueRange[] = [];
}
